openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -935,7 +935,11 @@ def test_extension_form_fields(self):
             }
         )
 
-    def test_register_form_third_party_auth_running_google(self):
+    @ddt.data(
+        True,
```

Comment:
Instead of a true/false switch for ddt, what about just specifying country code values?  Then we could include more codes as desired --> PK, pK, Pk, pk